cmd/res/configuration.toml

```diff
@@ -4,122 +4,24 @@ MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to me
 LogLevel = "INFO"
   # InsecureSecrets are required for when Redis is used for message bus
   [Writable.InsecureSecrets]
-    [Writable.InsecureSecrets.DB]
-    path = "redisdb"
-      [Writable.InsecureSecrets.DB.Secrets]
-      username = ""
-      password = ""
     [Writable.InsecureSecrets.MQTT]
     path = "credentials"
       [Writable.InsecureSecrets.MQTT.Secrets]
       username = ""
       password = ""
-  [Writable.Reading]
-  ReadingUnits = true
-  [Writable.Telemetry]
-  Interval = "30s"
-  PublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
-    [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
-    # Device SDK provided Service Metrics
-    EventsSent = false
-    ReadingsSent = false
-    # Common Security Service Metrics
-    SecuritySecretsRequested = false
-    SecuritySecretsStored = false
-    SecurityConsulTokensRequested = false
-    SecurityConsulTokenDuration = false
-    [Writable.Telemetry.Tags] # Contains the service level tags to be attached to all the service's metrics
-#    Gateway="my-iot-gateway" # Tag must be added here or via Consul Env Override can only chnage existing value, not added new ones.
+
 [Service]
-HealthCheckInterval = "10s"
-Host = "localhost"
 Port = 59982
-ServerBindAddr = ""  # blank value defaults to Service.Host value
 StartupMsg = "device mqtt started"
-# MaxRequestSize limit the request body size in byte of put command
-MaxRequestSize = 0 # value 0 unlimit the request size.
-RequestTimeout = "5s"
-
-[Registry]
-Host = "localhost"
-Port = 8500
-Type = "consul"
-
-[Clients]
-  [Clients.core-data]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59880
-
-  [Clients.core-metadata]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59881
 
 [MessageBus]
-Protocol = "redis"
-Host = "localhost"
-Port = 6379
-Type = "redis"
-AuthMode = "usernamepassword"  # required for redis messagebus (secure or insecure).
-SecretName = "redisdb"
   [MessageBus.Topics]
   PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name>/<source-name> will be added to this Publish Topic prefix
   CommandRequestTopic = "edgex/device/command/request/device-mqtt/#"   # subscribing for inbound command requests
   CommandResponseTopicPrefix = "edgex/device/command/response"   # publishing outbound command responses; <device-service>/<device-name>/<command-name>/<method> will be added to this publish topic prefix
```

Comment:
should those topics be removed as well?

Reply:
That is in the next PR. keeping breaking changes in separate PRs